sklearn/covariance/empirical_covariance_.py

```diff
@@ -79,6 +80,8 @@ def empirical_covariance(X, assume_centered=False):
     else:
         covariance = np.cov(X.T, bias=1)
 
+    if covariance.ndim == 0:
```

Comment:
So it is not really related to the issue, but to respect the docstring. I guess this is ok as a bugfix.